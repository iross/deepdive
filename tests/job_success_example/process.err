  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0100 45862  100 45862    0     0  14.1M      0 --:--:-- --:--:-- --:--:-- 21.8M
