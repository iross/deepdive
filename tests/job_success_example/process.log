000 (1906957.000.000) 03/26 08:28:50 Job submitted from host: <128.104.101.254:36127>
    DAG Node: job000002+job000002
...
001 (1906957.000.000) 03/26 08:29:24 Job executing on host: <128.104.55.47:56176>
...
006 (1906957.000.000) 03/26 08:29:33 Image size of job updated: 7
	2  -  MemoryUsage of job (MB)
	1604  -  ResidentSetSize of job (KB)
...
006 (1906957.000.000) 03/26 08:30:04 Image size of job updated: 289272
	2  -  MemoryUsage of job (MB)
	1604  -  ResidentSetSize of job (KB)
...
005 (1906957.000.000) 03/26 08:30:07 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:00:01, Sys 0 00:00:00  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:00:01, Sys 0 00:00:00  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	234804  -  Run Bytes Sent By Job
	138522  -  Run Bytes Received By Job
	234804  -  Total Bytes Sent By Job
	138522  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :   635162  1000000   3038579
	   Memory (MB)          :        2     1000      1000
...
