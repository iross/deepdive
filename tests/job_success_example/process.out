_CONDOR_ANCESTOR_3822=3850:1427118199:2696399109
_CONDOR_ANCESTOR_3850=16868:1427376563:2151094102
_CONDOR_ANCESTOR_16868=16872:1427376565:3203988300
TEMP=/var/lib/condor/execute/slot1/dir_16868
_CONDOR_SCRATCH_DIR=/var/lib/condor/execute/slot1/dir_16868
_CONDOR_SLOT=slot1_7
BATCH_SYSTEM=HTCondor
TMPDIR=/var/lib/condor/execute/slot1/dir_16868
_CONDOR_CHIRP_CONFIG=/var/lib/condor/execute/slot1/dir_16868/.chirp_config
_CONDOR_JOB_PIDS=
TMP=/var/lib/condor/execute/slot1/dir_16868
_CONDOR_JOB_AD=/var/lib/condor/execute/slot1/dir_16868/.job.ad
_CONDOR_JOB_IWD=/var/lib/condor/execute/slot1/dir_16868
_CHIRP_DELAYED_UPDATE_PREFIX=Chirp*
_CONDOR_MACHINE_AD=/var/lib/condor/execute/slot1/dir_16868/.machine.ad
PATH=/usr/local/bin:/usr/kerberos/bin:/usr/bin:/bin
MachineAdTest: looking for <IsEuclid>
Euclid test returned <no>
Calling curl to fetch chtcinnerwrapper try 0
Curl error <0>
MachineAdTest: looking for <IsEuclid>
Done Running chtcinnerwrapper Result was <0>
All went well!
